packages-content-model/roosterjs-content-model-core/lib/utils/paste/mergePasteContent.ts

```diff
@@ -45,12 +47,16 @@ export function mergePasteContent(
         defaultDomToModelOptions,
         {
             processorOverride: {
+                '#text': pasteTextProcessor,
                 entity: createPasteEntityProcessor(domToModelOption),
                 '*': createPasteGeneralProcessor(domToModelOption),
             },
             formatParserOverride: {
                 display: pasteDisplayFormatParser,
             },
+            additionalFormatParsers: {
+                container: [containerWidthFormatParser],
```

Comment:
Question: in the Paste plugin we add some parsers that are always added to the domToModel options.

Should we move those parsers here? I can do that if needed in another PR.

Reply:
Paste Plugin is an optional plugin. We use this plugin when we want to handle content copied from some special known sources such as Word. 

Here it is some common behavior that even without PastePlugin, they will still take effect.

Also, PastePlugin has the equal rights to other 3rd plugins, so the behavior of parsers added in PastePlugin may overwrite parsers added by other plugins, and is also able to be overwritten by other plugins, depends on the plugin order. Here it is the default behavior so it will not overwrite any other plugin's behavior, it can always be overwritten.